package.json

```
@@ -1,17 +1,34 @@
 {
-  "name": "",
-  "version": "",
-  "description": "",
-  "author": "",
-  "license": "",
+  "name": "reviews_module",
+  "description": "Reviews Module for TREK bikes Clone",
+  "main": "index.js",
+  "version": "1.0.0",
+  "author": "Hayden Bass",
+  "license": "UNLICENSED",
   "engines": {
     "node": ">=6.13.0"
   },
   "scripts": {
+    "react-dev": "webpack -d --watch",
+    "server-dev": "nodemon server/index.js",
+    "start": "nodemon server/index.js"
```

```suggestion
    "start": "node server/index.js"
```
Again, not sure if this matters, but I don't think the deployed version will need to monitor for changes in the server code.